docs/specs/validation/link-validation.yml

```diff
@@ -417,3 +417,36 @@ outputs:
     {"message_severity":"suggestion","code":"image-inline","message":"Image 'a.jpg' is inline with text.","file":"a.md","line":21,"end_line":21,"column":13,"end_column":18}
     {"message_severity":"suggestion","code":"image-inline","message":"Image 'a.jpg' is inline with text.","file":"a.md","line":21,"end_line":21,"column":44,"end_column":49}
 ---
+# Validate redirected link
+inputs:
+  docfx.yml: |
+    markdownValidationRules: rules.json
+  rules.json: |
+    {
+      "links": {
+        "rules": [
+          {
+            "type": "RedirectedLink",
+            "message": "Redirected link '{0}' will be broken in isolated environments. Replace with a relative link.",
+            "code": "redirected-link",
+            "severity": "SUGGESTION",
+            "exclusions": [
+              "includes",
+              "landingpage",
+              "hubpage",
+              "toc"
+            ],
+            "redirectedLinkTypes": [
+               "aka.ms", 
+               "FWLinks"
+            ]
+          }
+        ]
+      }
+    }
+  file1.md: |
+    [external link](https://go.microsoft.com/fwlink/p/?LinkId=389595)
```

Comment:
do we need to validate http link (<a href=x)? 
if so, how about add a this kind of test case?

Reply:
added